README.md

```diff
@@ -75,18 +75,22 @@ on: [push]
 
 jobs:
   compatibilitycheck:
+    permissions:
+      # needed if you set comment-pr to 'yes'
+      pull-requests: write
+      contents: read
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
         with:
-          node-version: '20'
+          node-version: '22'
       - name: Install dependencies
         run: npm install
       - name: Build plugin
         run: npm run build
       - name: Compatibility check
-        uses: grafana/plugin-actions/is-compatible@v1
+        uses: grafana/plugin-actions/is-compatible@main
```

Comment:
the v1 etc tagging does not really work properly, we might need a better concept of tags to work with different actions in this repository